src/transaction/log_manager.c

```diff
@@ -13920,7 +13910,15 @@ cdc_make_loginfo (THREAD_ENTRY * thread_p, LOG_LSA * start_lsa)
 	{
 	  if (total_length + consume->length + MAX_ALIGNMENT > cdc_Gl.consumer.log_info_size)
 	    {
-	      log_infos = (char *) realloc (log_infos, total_length + consume->length + MAX_ALIGNMENT);
+	      temp_log_infos = (char *) realloc (log_infos, total_length + consume->length + MAX_ALIGNMENT);
```

Comment:
I'll reconsider about this part

Reply:
@joohok Do you mean you want to review after reconsidering it?